packages/api/src/client/index.ts

```diff
@@ -1141,7 +1141,7 @@ export class AssertionRecord {
   }
 
   async delete(
-    params: Omit<ComAtprotoRepoDeleteRecord.InputSchema, 'collection'>,
+    params: Omit<ComAtprotoRepoDeleteRecord.QueryParams, 'collection'>,
```

Comment:
I think you may need to rebuild lex-cli to grab Paul's recent fix.  I _think_ the way this was written is correct.